source/features/clean-repo-tabs.tsx

```diff
@@ -116,7 +112,7 @@ async function initProjects(): Promise<void | false> {
 		return;
 	}
 
-	if (await canUserEditOrganization()) {
+	if (pageDetect.isUserTheOrganizationOwner()) {
```

Comment:
`canUserEditOrganization` was renamed in the package because it was incorrect, but this is not the same as `isUserTheOrganizationOwner`

- https://github.com/refined-github/github-url-detection/pull/87
- https://github.com/refined-github/refined-github/pull/4534#issuecomment-878059232

If anything, we need to add `canUserEditOrganization () => select.exists('.btn-primary[href$="repositories/new"]')`. We didn't back then because we needed elementReady, which isn't yet available:

- https://github.com/refined-github/github-url-detection/issues/85

Reply:
Thanks for explaining, I'm confused as I think if a user can edit organization, they are certainly a owner. Now that I take a look at the actual selector, I think the name should be something else. Being able to create a repo only means they have org-wide write access. Also, the existing `canUserEditRepo` is checking for a settings tab, which only exists for those who can manage the repo, not just write access.